task/dev3/EntryPoint.java

```diff
@@ -0,0 +1,38 @@
+package task.dev3;
+
+import task.dev3.User.*;
+import task.dev3.EnterUser.*;
+import task.dev3.Operations.InformationPrinter;
+
+import java.io.*;
+import java.util.*;
+
+/**
+ * Class is entry point to work.
+ */
+public class EntryPoint {
+  public static void main(String[] args) {
+    List<User> userList = new ArrayList<User>();
+    Scanner input = new Scanner(System.in);
+    int choose = 0;
+    int i = 0;
+    
+    while (i < 1) {
+      System.out.println("Do you want to add user?\n" + "1 - Yes\n" + "2 - No");
```

Comment:
why man/woman but 1 or for yes/no? Be consistent - use everywhere the same approach